kubernetes/hmda-platform/values.yaml

```diff
@@ -33,7 +36,7 @@ kafka:
 service:
   type: ClusterIP
   account:
-    name: hmda-service-account
+    name: hmda-service-account-default
```

Comment:
Do we need to add this `service-acount` change to the `values-dev.yaml` file?

Reply:
Added